nix/gecko_env.nix

```diff
@@ -60,8 +62,9 @@ in gecko.overrideDerivation (old: {
 
     # Build LDFLAGS and LIBRARY_PATH
     echo "export LDFLAGS=\"$NIX_LDFLAGS\"" >> $geckoenv
-    echo "export LIBRARY_PATH=\"$CMAKE_LIBRARY_PATH\"" >> $geckoenv
-    echo "export LD_LIBRARY_PATH=\"$CMAKE_LIBRARY_PATH\"" >> $geckoenv
+    echo "export SQLITE_LIB=${sqlite.out}/lib/" >> $geckoenv
```

Comment:
Is this environment variable really used by sqlite ?

Reply:
The environment variable does not change anything, it exists in order to increase the readability of the next two lines.

Reply:
Don't expose it then, `gecko-env` is fickle, and i'd like to introduce as little changes as possible there